pkg/lint/rules/template.go

```diff
@@ -45,7 +45,7 @@ var (
 )
 
 // Templates lints the templates in the Linter.
-func Templates(linter *support.Linter, values map[string]interface{}, namespace string, strict bool) {
+func Templates(linter *support.Linter, values map[string]interface{}, name, namespace string, strict bool) {
```

Comment:
more specific naming nit:

```suggestion
func Templates(linter *support.Linter, values map[string]interface{}, releaseName, namespace string, strict bool) {
```